src/common/InfoDashboard/index.vue

```diff
@@ -136,6 +133,8 @@ const CHART_COLORS = {
   [CHART_TYPE.circulingSupply]: '#FF7C03',
 }
 
+const REFERRAL_REWARD = 1 //%
```

Comment:
Is it okay to store this as const? Mb put it in `.env` for easier changes in production

Reply:
We discussed this, it can be a constant in a component